src/promisify.js

```diff
@@ -78,6 +78,18 @@ function promisify(orig, options) {
 }
 
 /**
+ * Promisify the given name on the given object, and create a copy of the object.
+ *
+ * @param {*} obj A value that can have properties.
+ * @param {String} methodName The method to promisify.
+ * @param {Boolean|String[]} options.variadic See the documentation for promisify.
+ * @return {Object} The promisified object.
+ */
+function promisifyMethod(obj, methodName, options) {
+  return promisifyMethods(obj, [methodName], options)[methodName];
```

Comment:
[This](https://github.com/mixmaxhq/promise-callbacks/pull/12/files#diff-1f0dd9d666794c01ded659349cb2512eR109) makes it look like you could just
```js
return promisify(obj[methodName].bind(obj), options);
```
, does that work?

Reply:
Sure that would work. But then you wouldn't get the error checking for

```
throw new TypeError('promisify.methods requires a truthy value');
```

found [here](https://github.com/mixmaxhq/promise-callbacks/pull/12/files#diff-1f0dd9d666794c01ded659349cb2512eR103).

My preference is to reduce duplicate code such that if we make more changes to `promisifyMethods` we don't have to remember to make those changes to `promisifyMethod`

Reply:
Yeah for me that concern is mitigated by having `promisifyMethods` basically share the proposed implementation for this method in the form of `promisify`.

Wouldn't it be kinda weird to have `promisify.method` emit an error talking about `promisify.methods`? Leaks implementation.

I don't feel strongly either way though.

Reply:
Makes sense. I went with your suggestion. Thanks!